source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -299,15 +400,36 @@ def _get_matrix(cls, json_response, origins, destinations):
 
         return sn_routing_matrix
 
-    def get_matrix_params(self, origins, destinations, mode, max_duration, request):
+    @classmethod
+    def _create_matrix_response_with_direct_path(cls, json_response, origin, destination, row):
+        routing = row.routing_response.add()
+        routes = json_response.get('response', {}).get('route', [])
+        if not routes:
+            return row
+        route = routes[0]
+        travel_time = route.get('summary', {}).get('travelTime')
+        if travel_time:
```

Comment:
can `travel_time` be 0?
`if travel_time is not None:`

Reply:
:+1: